src/main/java/com/iota/iri/storage/Tangle.java

```diff
@@ -206,7 +209,7 @@ private void updatePersistenceProvider(Persistable model, Indexable index, Strin
         }
     }
 
-    private void updateMessageQueueProvider(Persistable model, Indexable index, String item) {
+    public void updateMessageQueueProvider(Persistable model, Indexable index, String item) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Public method and constructor comments are required](https://app.codacy.com/manual/GalRogozinski/iri/pullRequest?prid=4896328)